src/components/refactor/dialogs/voltage-level-creation/bus-bar-section/bus-bar-section.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Grid } from '@mui/material';
+import React from 'react';
+import { BusBarSectionCreation } from './bus-bar-section-creation';
+import {
+    SWITCH_TYPE,
+    VOLTAGE_LEVEL_COMPONENTS,
+} from 'components/network/constants';
+import { BusBarSectionConnection } from './BusBarSectionConnection';
+import {
+    BUS_BAR_SECTIONS,
+    FROM_BBS,
+    HORIZONTAL_POSITION,
+    ID,
+    NAME,
+    SWITCH_KIND,
+    TO_BBS,
+    VERTICAL_POSITION,
+} from 'components/refactor/utils/field-constants';
+import ExpandableInput from 'components/refactor/rhf-inputs/expandable-input';
+
+export const BusBarSection = ({ id, type, errors }) => {
```

Comment:
I think we don't need to pass the "errors" object from the parent dialog to all the components